ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/CommandSubmissionCompletionIT.scala

```diff
@@ -101,14 +101,15 @@ final class CommandSubmissionCompletionIT extends LedgerTestSuite {
       wrongRequest = ledger.submitRequest(party, wrongExercise)
       failure <- ledger.submit(wrongRequest).mustFail("submitting an invalid choice")
     } yield {
-      assertGrpcError(
+      assertGrpcErrorRegex(
         failure,
         Status.Code.INVALID_ARGUMENT,
         Some(
           Pattern.compile(
             "(unknown|Couldn't find requested) choice " + badChoice
           )
         ),
+        checkDefiniteAnswerMetadata = true,
```

Comment:
Just making sure we are running the full compatibility test suite as well to ensure this won't break for older versions of the sdk.

Reply:
Not sure what you mean by that. Older SDK versions are excluded from compatibility tests, as they do not return the definite answer.